apps/electron/src/renderer/src/store/useWebsocket.ts

```diff
@@ -4,6 +4,7 @@ import type { SessionContext } from './useAuth'
 
 import { useLocalStorage, useWebSocket } from '@vueuse/core'
 import { defu } from 'defu'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
No need to replace in backend code I think. Or you should also add uuid package to the backend packages.